src/components/reports/ReportForm.tsx

```diff
@@ -0,0 +1,208 @@
+          <Field id="classification" name="variant.classification"/>
+          <ErrorMessage component="p"  className="error-text" name="variant.classification"/>
+          <label htmlFor="classificationEvidence">Classification Evidence</label>
+          <Field as="textarea" id="classificationEvidence" name="variant.classificationEvidence"/>
+          <ErrorMessage component="p"  className="error-text" name="variant.classificationEvidence"/>
+
+          <h2>Report</h2>
+          <label htmlFor="resultSummary">Result summary</label>
+          <Field as="textarea" id="resultSummary" name="result.resultSummary"/>
+          <ErrorMessage component="p"  className="error-text" name="result.resultSummary"/>
+          <label htmlFor="furtherTesting">Further testing</label>
+          <Field as="textarea" id="furtherTesting" name="result.furtherTesting"/>
+          <ErrorMessage component="p"  className="error-text" name="result.furtherTesting"/>
+          <label htmlFor="testMethodology">Test Methodology</label>
+          <Field as="textarea" id="testMethodology" name="result.testMethodology"/>
+          <ErrorMessage component="p"  className="error-text" name="result.testMethodology"/>
+          <label htmlFor="authorisingDate">Authorised date</label>
+          <Field id="authorisingDate" type="date" name="result.authorisingDate"/>
+          <ErrorMessage component="p"  className="error-text" name="result.authorisingDate"/>
+          <label htmlFor="authorisingScientist">Authorising scientist</label>
+          <Field id="authorisingScientist" name="result.authorisingScientist"/>
+          <ErrorMessage component="p"  className="error-text" name="result.authorisingScientist"/>
+          <label htmlFor="authorisingScientistTitle">Authorising scientist title</label>
+          <Field id="authorisingScientistTitle" name="result.authorisingScientistTitle"/>
+          <ErrorMessage component="p"  className="error-text" name="result.authorisingScientistTitle"/>
+          <label htmlFor="reportingDate">Reporting date</label>
+          <Field id="reportingDate" type="date" name="result.reportingDate"/>
+          <ErrorMessage component="p"  className="error-text" name="result.reportingDate"/>
+          <label htmlFor="reportingScientist">Reporting scientist</label>
+          <Field id="reportingScientist" name="result.reportingScientist"/>
+          <ErrorMessage component="p"  className="error-text" name="result.reportingScientist"/>
+          <label htmlFor="reportingScientistTitle">Reporting scientist title</label>
+          <Field id="reportingScientistTitle" name="result.reportingScientistTitle"/>
+          <ErrorMessage component="p"  className="error-text" name="result.reportingScientistTitle"/>
+
+          <br/>
+          <button type="submit">Submit</button>
+
+          {result !== "" && <textarea id="resultOutput" role="alert" rows={20} defaultValue={result}/>}
```

Comment:
Does the textarea need to be inside of the Form component? No difference either way, just curious.

Reply:
Ah so I think I originally had it inside of the Formik component it turns out that can only have a single child - could have moved it outside of formik but ended up forgetting about it. Will eventually remove it by the time we get to a more reasonable form